lib/css_to_emotion.re

```diff
@@ -604,7 +507,7 @@ let rec render_component_value =
     }
   | Unicode_range(_) => grammar_error(loc, "Unsupported unicode range")
   | [@implicit_arity] Function(f, params) => render_function(f, params)
-  | [@implicit_arity] Float_dimension(number, "ms", Time) when mode == Bs_css =>
+  | [@implicit_arity] Float_dimension(number, "ms", Time) =>
```

Comment:
The `implicit_arity` annotations are artifacts from refmt and could be removed. This could be replaced with `Float_dimension((number, "ms", Time))` (note double parents around tuple args).